docs/topics_5/web-openshift-troubleshoot.adoc

```diff
@@ -1,166 +1,44 @@
-
-*How to resolve it:*
-
-Perform either of the following:
-
-* Increase the quota for the OpenShift project. For additional information, see link:{OpenShiftDocsURL}/applications/quotas/quotas-setting-per-project.html[Quotas and Limit Ranges] and link:{OpenShiftDocsURL}/nodes/clusters/nodes-cluster-limit-ranges.html[Setting Limit Ranges] in the OpenShift documentation.
-* Reduce the requested resources for the {WebName} OpenShift project. It is recommended to have at least 2 CPUs and 4 GB of memory for the project.
-
-Once the request is within the available quota, attempt the deployment once again.
-
-.Pod takes longer than 600 seconds to become available
-
-After attempting a deployment, the `-deploy` Pods time out and report the following error.
-
-[source,options="nowrap",subs="+quotes"]
-----
-error: update acceptor rejected mta-web-console-executor-1: Pods for rc 'mta/<POD_NAME>' took longer than 600 seconds to become available
-----
-
-These errors appear after the Pods time out and are placed into an error state.
-
-*What it means:*
-
-The deployment Pods are unable to successfully launch the Pods.
-
-This error can be caused by a number of sources, with the following being the most common:
-
-* The OpenShift instance is currently out of resources to deploy the Pod in a timely manner.
-* The images were unable to be successfully pulled from the registry.
-
-*How to resolve it:*
-
-Attempt the deployment again, and view the logs and events of the non deployment Pods while they are being created. These messages will provide context to the underlying errors resulting in the deployment Pod timeouts.
-
-* If the OpenShift instance runs out of resources, link:{OpenShiftDocsURL}/nodes/clusters/nodes-cluster-resource-levels.html[analyze your cluster's capacity]. Once the capacity has increased, or there are fewer jobs executing, attempt the deployment once again.
-
-* If the images cannot be pulled from the registry, link:{OpenShiftDocsURL}/registry/accessing-the-registry.html[access the registry] to verify that the images are present and link:{OpenShiftDocsURL}/registry/accessing-the-registry.html#registry-viewing-logs_accessing-the-registry[check the logs].
+// Common issues
+// include::topics_5/proc_web-troubleshooting-common-issues.adoc[leveloffset=+2]
 
 == Reporting issues
 
-The {ProductName} uses JIRA as its issue tracking system. If you encounter any issues while using the {WebName}, please file a JIRA Issue by following the below instructions.
+The {ProductName} uses Jira as its issue tracking system. If you encounter any issues while using the {WebName}, please file a Jira issue by following the instructions below:
 
-NOTE: If you do not have one already, you must sign up for a JIRA account in order to create a JIRA issue.
+NOTE: If you do not already have a Jira account, you must sign up for one in order to create a Jira issue.
```

Comment:
First clause is fluff. Suggested rewrite: "You must sign up for a Jira account in order to create an issue."